crates/bevy_ecs/src/storage/blob_vec.rs

```diff
@@ -437,6 +437,12 @@ impl BlobVec {
             }
         }
     }
+
+    /// Get the dropper for this vector. Used if caller has type-erased pointer
```

Comment:
The docs here need an editing pass for clarity.

Reply:
Rewrote the doc, is the new version better?